test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```

Comment:
you don't need do test 
please delete this file

Reply:
not fix yet

Reply:
not fix yet

Reply:
not fix yet